react/features/chat/reducer.js

```diff
@@ -48,12 +56,20 @@ ReducerRegistry.register('features/chat', (state = DEFAULT_STATE, action) => {
             messages: []
         };
 
+    case SET_PRIVATE_MESSAGE_RECIPIENT:
+        return {
+            ...state,
+            isOpen: action.participant ? true : state.isOpen,
+            privateMessageRecipient: action.participant
+        };
+
     case TOGGLE_CHAT:
         return {
             ...state,
             isOpen: !state.isOpen,
             lastReadMessage: state.messages[
-                navigator.product === 'ReactNative' ? 0 : state.messages.length - 1]
+                navigator.product === 'ReactNative' ? 0 : state.messages.length - 1],
+            privateMessageRecipient: state.isOpen ? undefined : state.privateMessageRecipient
```

Comment:
Why override privateMessageRecipient? To clear it on close, right?

Reply:
Yep